audioread/macca.py

```diff
@@ -323,6 +328,11 @@ def read_data(self, blocksize=4096):
             blob = data[:size]
             yield blob
 
+    def seek(self, pos):
+        """Seeks to the position in the file"""
```

Comment:
Style nitpicking: it's nice to write docstrings (a) in the imperative voice, and (b) as complete sentences, including a period. So I suggest:

> Seek to a position in the file.